Core/HistoryManager.swift

```diff
@@ -244,3 +228,59 @@ class HistoryStoreEventMapper: EventMapping<HistoryStore.HistoryStoreEvents> {
         fatalError("Use init()")
     }
 }
+
+extension HistoryManager {
+
+    /// Should only be called once in the app
+    public static func make(isAutocompleteEnabledByUser: @autoclosure @escaping () -> Bool,
```

Comment:
Moved creation logic here.  These auto closures mean I don't need to pass in AppSettings into a Core class.